docs/modules/user-guide/partials/proc_building-devfile-registry.adoc

```diff
@@ -0,0 +1,47 @@
+[id="building-custom-devfile-registry_{context}"]
+= Building a devfile registry
+
+[role="_abstract"]
+Make your repository into a devfile registry by building a `devfile index` container image. Deploy the `devfile index` image and the `Open Containers Initiative (OCI) registry image` to your cloud or cluster to form the devfile registry.
```

Comment:
I would leave out the "OCI registry image" part here, as it's unlikely the user will need to worry about that.